scripts/download_webcams.sh

```diff
@@ -6,6 +6,6 @@ set -x
 1>&2 echo 'Downloading webcam images'
 lftp -e "mirror -c --parallel=$IPL_WEBCAM_WORKER --verbose / /var/webcam; quit;" -u $IPL_WEBCAM_USER,$IPL_WEBCAM_PASSWORD $IPL_WEBCAM_SERVER
 # Delete all old webcam images
-find /var/webcam -mtime +$IPL_WEBCAM_KEEP_DAYS -type f -name '*.jpeg' -delete
+find /var/webcam -mtime +$IPL_WEBCAM_KEEP_DAYS -type f -name '*.jpg' -delete
```

Comment:
Potentially, this deletes less (`*.jpg`) than the download script downloads (`*`), right?

I'd rather solve this problem once and for all by going with `'*'`.